test/gtest/ucp/ucp_test.h

```diff
@@ -231,6 +231,7 @@ class ucp_test : public ucp_test_base,
     void flush_ep(const entity &e, int worker_index = 0, int ep_index = 0);
     void flush_worker(const entity &e, int worker_index = 0);
     void flush_workers();
+    ucp_test::entity* find_entity_by_ep(ucp_ep_h ep);
```

Comment:
remove `ucp_test::`
also maybe return entity& to match other functions?

Reply:
actually can't use entity& since we sometimes return null

Reply:
yes, can't return reference to `entity&`

Reply:
how about removing `ucp_test::`?

Reply:
done